.circleci/config.yml

```diff
@@ -48,8 +48,8 @@ commands:
 
   checkout-charts:
     description: |
-      Checkout G-Research charts repo so we can update them later
-      Checkout from: https://github.com/G-Research/charts
```

Comment:
I don't think this is correct.  I don't believe we migrated charts yet.

Reply:
Thanks - fixed in commit 682f2c4d.